onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -528,6 +554,12 @@ and present state are optional. Present state could appear in output even when p
           "past state for key and value with shape (2, batch_size, num_heads, past_sequence_length, head_size).",
           "T3",
           OpSchema::Optional)
+      .Input(
+          9,
+          "extra_add",
+          "additional add to QxK' with shape (1, num_heads, sequence_length, sequence_length).",
```

Comment:
shape is (batch_size, ...)?

Reply:
will update with batch_size like in Attention(), will make this change as part of another PR though,